Cargo.toml

```diff
@@ -81,4 +83,4 @@ regtest-bitcoin = ["regtest-node" , "rpc"]
 regtest-electrum = ["regtest-node", "electrum", "electrsd/electrs_0_8_10"]
 #TODO: Check why esplora in electrsd isn't working.
 #regtest-esplora-ureq = ["regtest-node", "esplora-ureq", "electrsd/esplora_a33e97e1"]
-#regtest-esplora-reqwest = ["regtest-node", "esplora-reqwest", "electrsd/esplora_a33e97e1"]
+#regtest-esplora-reqwest = ["regtest-node", "esplora-reqwest", "electrsd/esplora_a33e97e1"]
```

Comment:
I wonder why your editor removes newlines at the end of the files, while mine adds them automatically 😂

Reply:
Ah thanks.. I don't think its my editor.. I probably did it myself by mistake.. Reverted..